### PR TITLE
Warn user on file not created by jupyterdrive

### PR DESCRIPTION
**Pull request by Carreau**
Should at least be of (some) help.

### CODE REVIEW

File: jupyterdrive/gdrive/drive-contents.js [hunk 1]
```diff
@@ -116,6 +116,18 @@ define(function(require) {
             model['content'] = notebook_model.notebook_from_file_contents(contents);
             model['writable'] = metadata['editable'];
             return model;
+        }).catch(function(data){   
+            var reason ='Unknown Error.';
+            if( data.xhr.status === 404){
+                reason = 'We cannot access requested resource\n'+
+                         'This can happend if the resource was not created with jupyter drive.\n'+
```
Comment (KesterTong) [inline jupyterdrive/gdrive/drive-contents.js]: happend -> happen

File: jupyterdrive/gdrive/drive-contents.js [hunk 2]
```diff
@@ -116,6 +116,18 @@ define(function(require) {
...
+                         'Please re-upload the resource through jupyter-drive and re-try.';
```
Comment (KesterTong) [inline jupyterdrive/gdrive/drive-contents.js]: This isn't clear own what the user should do.  How about "Please re-upload the resource by dragging a copy of the file onto the Jupyter file manager".  Dragging a file into tree view doesn't actually work yet, but we can fix it.

**Comment by KesterTong** [comment]
Looks good, but I think it would be even better to catch the error at line 110.  They way you've written it, we catch all errors, but we really only want to catch errors from the specific call made at line 110.

**Comment by Carreau** [comment]
Moved the `.catch()`  and updated message.

**Comment by KesterTong** [comment]
just move the ```.catch()``` one more line upwards again...

The reason is that currently catch will be executed when any error happens during contents_prm.  But if we move this upwards (to line 110, inside ```metadata_prm.then( ... )```), it will only get executed when metadata_prm is successful, but ```api_utils.download(resource['downloadUrl'])``` fails.  Which is exactly when we want this code to execute.

**Comment by Carreau** [comment]
Will do, brain overheat right now. I'm getting confused with the promise all and contents_prm being cain with metadata ...

**Comment by Carreau** [comment]
> We cannot access requested resource\n -> We cannot access requested resource. \n

Done.

**Comment by KesterTong** [comment]
great, thanks.

**Pull request merged**